indra/explanation/pathfinding/pathfinding.py

```diff
@@ -263,7 +354,8 @@ def bfs_search(g, source_node, reverse=False, depth_limit=2, path_limit=None,
                             ign_vals = None
                             pass
                         else:
-                            ign_vals = yield new_path
+                            ign_vals = yield new_path, \
+                                       ['N/A'] * (len(new_path) - 1)
```

Comment:
It seems like the function is changed to generate a tuple of a path + ignore values instead of just a path? Does it do that always or just in some condition? Can you update the documentation? Also this function is called in `bfs_search_multiple_nodes` which currently only expects paths be generated, function call has to be updated there.